feat: add Yung-Qi & Thomas to educators (#561)
dwh-outsite/dwhdelft.nl: content/educators/yung-qi.md

---
name: Yung-Qi
pronouns_en: he/him
pronouns_nl: hij/hem
quote:

>   Ik probeer niemand te overtuigen om anderen te accepteren die zich zelf zijn, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zoals die zijn.

Reviewer: ```suggestion
  Ik wil niemand overtuigen, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zoals die zijn.
```
(oid)

Anders wordt ie wel erg lang
![CleanShot 2024-04-11 at 19 34 57@2x](https://github.com/dwh-outsite/dwhdelft.nl/assets/15815208/bf8be083-3836-4748-83be-4a4309dd98e0)

Author: Ik heb er nu 'Ik wil niemand te overtuigen om anderen te accepteren, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zoals ze zijn.' van gemaakt, iets meer true to the quote, en even lang als Dante's quote

>   Ik probeer niemand te overtuigen om anderen te accepteren die zich zelf zijn, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zo als die zijn.

Reviewer 2: ```suggestion
  Ik probeer niemand te overtuigen om anderen te accepteren die zich zelf zijn, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zoals die zijn.
```

Author: Ik ga even aan Yung vragen of dit prima is, het is immers een quote ;)

text:

>   Hey, mijn naam is Yung-Qi 24 jaar en ben werk voor een onafhankelijke milieu advies bureau. Ik geef al een tijdje voorlichtingen. Dit doe ik omdat ik overtuigd ben dat als ik dit zelf had gehad in mijn middelbare school tijd, dat het mij waarschijnlijk wel veel had geholpen. Zelf ben ik een transman die vrij wel alles zelf moest uitzoeken en ontdekken. ik help graag andere met hun zoek tocht naar hun eigen identiteit en sexualiteit en wil het graag voor hun een stukje makkelijker / toegankelijker maken.

Reviewer 2: ```suggestion
  Hey, mijn naam is Yung-Qi 24 jaar en ben werkzaam voor een onafhankelijk milieu advies bureau. Ik geef al een tijdje voorlichtingen. Dit doe ik omdat ik overtuigd ben dat als ik dit zelf had gehad in mijn middelbare schooltijd, dat het mij waarschijnlijk wel veel had geholpen. Zelf ben ik een transman die vrijwel alles zelf moest uitzoeken en ontdekken. Ik help graag anderen met hun zoektocht naar hun eigen identiteit en sexualiteit en wil het graag voor hen een stukje makkelijker/toegankelijker maken.
```
